interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer: MVC stands for model, view, control.
 
-  Researched answer:
+  Researched answer: Rails likes to separte the idea of an applications data, user interface and control logic. They have seperate files to handle each part of the pattern.
 
 
 
 2. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: Relational databases are databases that have aceess to data that is related to them.
 
-  Researched answer:
+  Researched answer: Relational databases are collections of data and items with a pre-defined relationship between them.
 
 
 
 3. What is object-relational mapping?
 
-  Your answer:
+  Your answer: Not really sure.
```

Comment:
Okay, so week five - we have heard a number of times that not being sure isn't an acceptable answer for the interview questions. If you need to answer a similar question or a simpler question or relate the info back to something you are familiar with thats what this space is for. I am marking this assessment as incomplete until you push up a version of this question with an answer you could give in an interview.